sources/frontend.c

```diff
@@ -722,7 +724,7 @@ static od_frontend_status_t od_frontend_remote_client(od_relay_t *relay,
 		server->is_copy = 0;
 		break;
 	case KIWI_FE_QUERY:
-		if (instance->config.log_query)
+		if (instance->config.log_query && route->rule->log_query)
```

Comment:
```instance->config.log_query || route->rule->log_query```  ?

Reply:
Hello. I did not intend to include the second commit in this PR. However the second is to give the possibility to log queries on a per route configuration. So in order to to log queries for a given route one would have to set log_query both in the global config and in the route itself.

Reply:
May be || is better than &&

Reply:
let's change here the condition of disjunction into conjunction and LGTM?